src/hipscat_import/arguments.py

```diff
@@ -233,6 +234,7 @@ def from_params(
         pixel_threshold=1_000_000,
         debug_stats_only=False,
         filter_function=None,
+        schema_file=None,
```

Comment:
Should the schema_file input be added to the `from_command_line` method as well?